reddit.js

```diff
@@ -2,4 +2,49 @@ var reddit = require("./data.json")
 
 // Write your solutions below
 
-console.log(reddit["data"]);
\ No newline at end of file
+var children = reddit.data.children;
+// var childTitles = children[0].data.title;
+// var childLinks = children[0].data.permalink;
+// var childText = children[0].data.selftext;
+// var points = children[0].data.score;
+
+// console.log(childTitles);
+
+
+// QUESTION 1
+children.forEach(function(titles) {
```

Comment:
Here, a more appropriate name for your parameter instead of titles would be 'child', as the forEach iterator is going to pick up on each child object belonging to the children array.